src/sentry/api/endpoints/auth_config.py

```diff
@@ -57,8 +61,7 @@ def respond_authenticated(self, request: Request):
         next_uri = self.get_next_uri(request)
 
         if not is_valid_redirect(next_uri, allowed_hosts=(request.get_host(),)):
-            active_org = self.get_active_organization(request)
-            next_uri = get_org_redirect_url(request, active_org)
+            next_uri = get_org_redirect_url(request, self.active_organization)
```

Comment:
So you can see here how that works out -- prev it was a get, now it's a property.  Again, this avoids the giant hack that was necessary to protect against the side effects in the function below.